webapp/components/topbar/Topbar.js

```diff
@@ -85,6 +86,27 @@ class Topbar extends Component {
         }
         break;
       }
+      case 'handleTutorial': {
+        const [action, payload] = click.parameters;
+        if (this.props.modelState === MODEL_STATE.INSTANTIATED || this.props.modelState === MODEL_STATE.SIMULATED) {
+          this.props.openConfirmationDialog({
+            title: 'Warning',
+            message: 'A NetPyNE model has already been instantiated or simulated.'
```

Comment:
Let's put this message into a global variable DEFAULT_CONFIRMATION_DIALOG_MESSAGE as it been re-used.